api/lanes.py

```diff
@@ -571,6 +572,7 @@ def url_arguments(self):
             languages=self.language_key,
             audiences=self.audience_key
         )
+        kwargs[Facets.ORDER_FACET_GROUP_NAME] = Facets.ORDER_SERIES_POSITION
```

Comment:
This is what the related books uses to make a link to the series lane. This makes it link to the feed sorted by series position.